### DISCUSSION TITLE
비글챌 45회차 반성문

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#474] [6. 반성문]
안녕하세요? 오랜만에 반성문을 쓰게 됐네요.. 

지난 주에 중간고사 벼락치기와 자소서 작성으로 인해 글쓰기를 미루다가 까먹어버렸습니다. 죄송합니다.

이번 주부터는 진짜로 열심히 글 쓰겠습니다!!

여담이지만 요즘 너무 나태해진 것 같아서 5월에는 정처기와 오픽을 신청해봤습니다!! 

다들 시험기간인데도 열심히 개발하고.. 글 쓰고.. 하시는 모습들을 보며 자극 받고 있습니다

다들 화이팅입니다~~✨

**Comment by 02ggang9** [comment]
화이팅입니다~
